egs/wsj/s5/steps/nnet3/multilingual/allocate_multilingual_examples.py

```diff
@@ -0,0 +1,209 @@
+#!/usr/bin/env python
```

Comment:
@danpovey 
This python script  reads lang2len as input and it randomly selects new language w.r.t probability of remaining examples in each language and it outputs ranges.* with format 
<lang-id> <start-scp-line> <num-example>
and then the bash file get_egs.sh generates egs.*.scp w.r.t generated ranges.* in parallel.
The reason that I didn't read scp in python was that I should load whole scp file for each language in memory, that is not efficient!
Should I change the process to stream the scp files in python line by line (batch of minibatch-size each time!!)?